add instructions about SARIF file too large (#27990)
DataDog/documentation: content/en/security/code_security/troubleshooting/_index.md

### Disabling static repository scanning
To disable scanning Static Code Analysis (SAST) or static Software Composition Analysis:
- If you are scanning GitHub repositories through Datadog-hosted scanning, navigate to [**Code Security > Setup**][17], click **Enable scanning for your repositories**, and disable the toggles previously enabled for scanning either all connected repositories or each repository.
- If you are scanning source code repositories through your CI pipelines, remove the relevant job(s) from your CI pipelines.

### Disabling runtime SCA on your services

> SCA can be enabled on your running services using two methods: the UI or manually using the `DD_APPSEC_SCA_ENABLED` environment variable. When you disable SCA, you must use the *same method* you used to enable SCA.

Reviewer: ```suggestion
SCA can be enabled on your running services using one of the following two methods: 
- The Datadog UI. 
- Manually, using the `DD_APPSEC_SCA_ENABLED` environment variable.

To disable SCA, you must use the *same method* you used to enable SCA.
```
